### PR TITLE
Update `schedule`, `testimonial`, `venue` and `what's included` docs

### PR DESCRIPTION
**Pull request by geoffreymm**
Bulk update merchandise fields. There are still a couple to tackle:
- Highlights
- Trip tips

### CODE REVIEW

**Comment by geoffreymm** [comment]
[Update drops documentation on easol.dev](https://trello.com/c/glhZWKqz/53-update-drops-documentation-on-easoldev)

File: docs/reference/objects/product/index.md [hunk 1]
```diff
@@ -153,15 +153,15 @@ Returns the sum of remaining stock for a products variants, if any of the produc
 
 # product.schedule _(deprecated)_
 
-Returns an array of the products [schedule elements]({% link docs/reference/objects/product/schedule_element.md %})
+Returns an array of the products [schedule elements]({% link docs/reference/objects/product/schedule_element.md %}).
```
Comment (ianmooney) [inline docs/reference/objects/product/index.md:156]: Can we add an apostrophe here too - `array of the product's`

File: docs/reference/objects/product/index.md [hunk 2]
```diff
@@ -224,7 +224,7 @@ Returns the product's [venue]({% link docs/reference/objects/product/venue.md %}
 
 # product.whats_included
 
-Return the product [whats_included]({% link docs/reference/objects/product/whats_included.md %})
+Return the produc's [`whats included`]({% link docs/reference/objects/product/whats_included.md %})
```
Comment (ianmooney) [inline docs/reference/objects/product/index.md:227]: Typo here - `product's` and `what's`.

File: docs/reference/objects/product/whats_included.md [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
 parent: Product
 ---
 
-The `whats_included` object have the following attributes
+The `whats_included` object have the following attributes:
 
 # whats_included.included
 
-Return and array of words with included elements
+Return and array of words with included elements.
```
Comment (ianmooney) [inline docs/reference/objects/product/whats_included.md:11]: Typo here `Return an array`

How about the following:

```
Returns an array of strings representing what’s included.
```

File: docs/reference/objects/product/whats_included.md [hunk 4]
```diff
@@ -1,16 +1,15 @@
...
 
 # whats_included.not_included
 
-Return an array highlighting not included elements 
-
+Return an array highlighting not included elements .
```
Comment (ianmooney) [inline docs/reference/objects/product/whats_included.md:15]: How about the following:

```
Returns an array of strings representing what’s _not_ included.
```

**Review by ianmooney — APPROVED** [review]
Looks good - just left a few small comments.

File: docs/reference/objects/product/whats_included.md (hunk 4 above)
Comment (geoffreymm) [reply docs/reference/objects/product/whats_included.md]: Sounds good!

**Pull request merged**